### PR TITLE
[zh-cn]sync extend-resources/mutating-webhook-configuration-v1.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md [hunk 1]
```diff
@@ -234,6 +234,106 @@ MutatingWebhookConfiguration 描述准入 Webhook 的配置，该 Webhook 可接
...
+  - **webhooks.matchConditions** ([]MatchCondition)
+
+    **补丁策略：根据 `name` 键执行合并操作**
+
+    **映射：键 `name` 的唯一值将在合并过程中保留**
+
+    <!--
+    MatchConditions is a list of conditions that must be met for a request to be sent to this webhook. Match conditions filter requests that have already been matched by the rules, namespaceSelector, and objectSelector. An empty list of matchConditions matches all requests. There are a maximum of 64 match conditions allowed.
+    -->
+    matchConditions 是将请求发送到此 webhook 之前必须满足的条件列表。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md:253]: 253 前面加空行，避免排版问题。下同。

File: content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md [hunk 2]
```diff
@@ -234,6 +234,115 @@ MutatingWebhookConfiguration 描述准入 Webhook 的配置，该 Webhook 可接
...
+      - 'object' - 来自传入请求的对象。对于 DELETE 请求，该值为 null。
+      - 'oldObject' - 现有对象。对于 CREATE 请求，该值为 null。
+      - 'request' - 准入请求的属性(/pkg/apis/admission/types.go#AdmissionRequest)。
+      - 'authorizer' - CEL 授权者。可用于对请求的主体（用户或服务帐户）执行授权检查。
+
+        <!--
+        See https://pkg.go.dev/k8s.io/apiserver/pkg/cel/library#Authz
+        -->
+        
+        参阅：https://pkg.go.dev/k8s.io/apiserver/pkg/cel/library#Authz
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md:311]:
```suggestion
        参阅： https://pkg.go.dev/k8s.io/apiserver/pkg/cel/library#Authz
```

File: content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md [hunk 3]
```diff
@@ -234,6 +234,115 @@ MutatingWebhookConfiguration 描述准入 Webhook 的配置，该 Webhook 可接
...
+      'authorizer.requestResource' - A CEL ResourceCheck constructed from the 'authorizer' and configured with the
+        request resource.
+      Documentation on CEL: https://kubernetes.io/docs/reference/using-api/cel/
+  
+      Required.
+      -->
+      
+      - 'authorizer.requestResource' - CEL ResourceCheck 从"授权方"构建并配置请求资源。
+  
+      CEL 文档：https://kubernetes.io/zh-cn/docs/reference/using-api/cel/
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md:323]:
```suggestion
      CEL 文档： https://kubernetes.io/zh-cn/docs/reference/using-api/cel/
```
需要增加一个空格或换行

**Review by windsonsea — COMMENTED** [review]
See [preview](https://deploy-preview-41121--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1/).

**Pull request merged**